colors/whitescale.vim

```diff
@@ -80,6 +80,10 @@ hi Underlined term=underline gui=underline ctermfg=fg guifg=fg
 hi VertSplit ctermbg=NONE ctermfg=NONE cterm=NONE guibg=NONE guifg=NONE gui=NONE
 hi WildMenu term=bold cterm=bold gui=bold ctermfg=fg ctermbg=bg guifg=fg guibg=bg
 
+hi vimHiKeyList ctermfg=fg ctermbg=NONE
```

Comment:
You'll want to add the corresponding settings for `gui` too.

What do these highlight groups do? Can you document that in the PR description?

Reply:
I also don't have any of these highlight groups, which makes me think they are non-standard.

Maybe they fit better in ones personal vim config?

Reply:
I honestly don't know. I used https://github.com/gerw/vim-HiLinkTrace to find these. I believe this one was used for the `=` signs in this file. The others were used by _some_ of the color values.